ocrd_tesserocr/deskew.py

```diff
@@ -100,11 +100,20 @@ def process(self):
                     # (we will overwrite @orientation anyway,)
                     # abort if no such image can be produced:
                     feature_filter='deskewed' if oplevel == 'page' else '')
-                if page_image_info.resolution != 1:
+                if self.parameter['dpi'] > 0:
+                    dpi = self.parameter['dpi']
+                    LOG.info("Page '%s' images will use %d DPI from override", page_id, dpi)
```

Comment:
same as above. "parameter" or "parameter override" is preferable.

Reply:
Chose the latter and force-pushed.